.github/workflows/ci-analytics.yml

```diff
@@ -15,6 +15,11 @@ jobs:
   lint-test:
```

Comment:
Putting this here but it's really for the overall PR: your PR explanation is extremely thorough and helpful, well done!